### PR TITLE
Documentation: move installation instructions for developers to page for developers

### PR DESCRIPTION
**Pull request by kolibril13**
## Oneline Summary of Changes

```
- added installation for devs (:pr:`PR NUMBER HERE`)
```

## Acknowledgements
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [x] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

**Comment by kolibril13** [comment]
Or maybe we can also move this last part from the installation instruction here:
https://docs.manim.community/en/latest/installation/for_dev.html
and have there to subchapters:
## Manim Installation For Developers with pip
## Manim Installation For Developers with poetry

**Comment by leotrs** [comment]
"For contributors" and "for developers" are meant to be separate :thinking:

**Comment by kolibril13** [comment]
> As for your comment, "For contributors" and "for developers" are meant to be separate

What would be the difference of these two?

**Comment by leotrs** [comment]
In theory, there are many more possible ways of contributing than just code/development. If in practice, we don't have much of a difference for now then it's fine to use either.

**Comment by kolibril13** [comment]
Do we want to put the pip chapter then to https://docs.manim.community/en/latest/installation/for_dev.html
?

**Comment by leotrs** [comment]
Sure

**Review by huguesdevimeux — APPROVED** [review]

File: docs/source/installation/for_dev.rst [hunk 1]
```diff
@@ -3,8 +3,38 @@ Manim Installation For Developers
...
-Installing Poetry
-*****************
+
+
+Manim Installation For Developers with pip
+******************************************
+
+If you'd like to contribute to and/or help develop ``manim-community``, you can
+clone this repository to your local device.  To do this, first make sure you
+have ``git`` installed.
```
Comment (behackl) [inline docs/source/installation/for_dev.rst:13]: I think `manim-community` should be reworded, the library is called `manim` by now, and nothing in or related to the library is actually called `manim-community`.

```suggestion
If you'd like to contribute to and/or help develop the community maintained
version of ``manim``, you can clone this repository to your local device.  To do this,
first make sure you have ``git`` installed.
```

**Review by behackl — APPROVED** [review]
LGTM, one minor nitpick.

**Pull request merged**